examples/fire_alarm_demo.py

```diff
@@ -25,6 +25,7 @@ class AlarmDemo(K1):
 
     async def async_demo1(self) -> None:
         """Main routine to demonstrate the API code."""
+        #logging.basicConfig(level=logging.DEBUG)
```

Comment:
I suppose this line can be removed?

Reply:
```suggestion
```

Reply:
Yes, added and commented it for newbies like myself; it gave me a bit more insights while debugging